felix/bpf-gpl/conntrack.h

```diff
@@ -294,6 +294,7 @@ static CALI_BPF_INLINE bool skb_icmp_err_unpack(struct cali_tc_ctx *ctx, struct
 
 	if (skb_refresh_validate_ptrs(ctx, ICMP_SIZE + sizeof(struct iphdr) + 8)) {
 		ctx->fwd.reason = CALI_REASON_SHORT;
+		INC(ctx, ERR_SHORT_PKTS);
```

Comment:
I wonder if we could merge these. We do not really use the reasons for anything except some debugging. And the reasons could be not funny numbers like now, but something that would work for error counters too.  So that we do  not need to remember to set both.

Maybe `DENY_REASON(ctx, xyz)`

Reply:
I was thinking do to so in next step. It seem the pattern for setting reason and counter is repeated many places. So definitely we can do so.